model-optimizer/mo/main.py

```diff
@@ -249,9 +249,17 @@ def emit_ir(graph: Graph, argv: argparse.Namespace):
 
     if not (argv.framework == 'tf' and argv.tensorflow_custom_operations_config_update):
         output_dir = argv.output_dir if argv.output_dir != '.' else os.getcwd()
-        print('\n[ SUCCESS ] Generated IR version {} model.'.format(get_ir_version(argv)))
-        print('[ SUCCESS ] XML file: {}.xml'.format(os.path.join(output_dir, argv.model_name)))
-        print('[ SUCCESS ] BIN file: {}.bin'.format(os.path.join(output_dir, argv.model_name)))
+
+        orig_model_name = os.path.join(output_dir, argv.model_name)
```

Comment:
To get rid of this annoying `.` in 
```
[ SUCCESS ] XML file: C:\Users\estepyre\Desktop\git\openvino\model-optimizer\.\model.xml
[ SUCCESS ] BIN file: C:\Users\estepyre\Desktop\git\openvino\model-optimizer\.\model.bin
```

```suggestion
        orig_model_name = os.path.normpath(os.path.join(output_dir, argv.model_name))
```